fastexcel-writer/src/main/java/org/dhatim/fastexcel/AlternateShading.java

```diff
@@ -15,22 +15,11 @@
  */
 package org.dhatim.fastexcel;
 
-import java.io.IOException;
-
 /**
  * Helper class to define a simple shading to alternate rows in a range of
  * cells.
  */
-class AlternateShading {
-
-    /**
-     * Range where alternate rows are shaded.
-     */
-    private final Range range;
-    /**
-     *  Index of cached fill pattern for shaded rows.
-     */
-    private final int fill;
+class AlternateShading extends Shading{
```

Comment:
I cannot see any other extension to `Shading` or direct usage of `Shading`. Is this new `Shading` class necessary?

Reply:
In our reports we are always coloring every third row since it is the optimal way for some longer reports.
It could also be, I guess, just a function in the AlternateShading class.

Like I previously mentioned, 
shading rows (extension of shadeAlternateRows) so that each Nth row is colored
[ instead of shadeAlternateRows(color), simply use: shadeRows(color, eachNRows) ]

Reply:
OK. I did not read carefully the code, the `Shading` is indeed used in method `shadeRows()`. Thanks for contributing!